code/mob/living.dm

```diff
@@ -1789,6 +1789,11 @@
 							// if the storage object contains mobs, use its p_class (updated within storage to reflect containing mobs or not)
 							if (locate(/mob) in A.contents)
 								. *= lerp(1, max(A.p_class, 1), mob_pull_multiplier)
+							else if (locate(/obj/item/gang_loot) in A.contents)
+								. *= lerp(1, max(A.p_class, 1), mob_pull_multiplier)
+						else if (A.always_slow_pull)
+							. *= lerp(1, max(A.p_class, 1), mob_pull_multiplier)
```

Comment:
Personally I fervently dislike this, but refactoring slowing the pulling of objects to use a more sane system than a tree of `if else`s and `locate() in`s is outside of the scope of this PR.